cartridges/int_twilio/cartridge/client/default/js/product/detail.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+var base = require('base/product/detail');
+
+base.updateAttribute = function () {
+    $('body').on('product:afterAttributeSelect', function (e, response) {
+        if (response.data.product.available) {
+            $('.add-to-cart').removeClass('d-none');
+            $('.product-subscription').addClass('d-none');
+        } else {
+            $('.add-to-cart').addClass('d-none');
+            $('.product-subscription').removeClass('d-none');
+        }
+
+        $('.subscriptionProductId').val(response.data.product.id);
+
+        if ($('.product-detail>.bundle-items').length) {
+            response.container.data('pid', response.data.product.id);
+            response.container.find('.product-id').text(response.data.product.id);
+        } else if ($('.product-set-detail').eq(0)) {
+            response.container.data('pid', response.data.product.id);
+            response.container.find('.product-id').text(response.data.product.id);
+        } else {
+            $('.product-id').text(response.data.product.id);
+            $('.product-detail:not(".bundle-item")').data('pid', response.data.product.id);
+        }
+    });
+}
+
+base.subscribe = function () {
+    $(document).ready(function() {
+        $('.subscriptionForm').submit(function (e) {
+            e.preventDefault();
+
+            var formData = $(this).serialize();
+
+            $.spinner().start();
+
+            $.ajax({
+                url: $(this).attr('action'),
+                type: 'post',
+                data: formData,
+                success: function (data) {
+                    $.spinner().stop();
+
+                    if (data.error) {
+                        $('.add-to-cart-messages').append(
```

Comment:
Try avoid building HTML with JS. Instead can have a hidden markup within the template which to be used as pattern.

Reply:
Fixed